cypress/support/index.d.ts

```diff
@@ -2,7 +2,10 @@
 
 declare namespace Cypress {
   interface Chainable<Subject> {
-    getByDataCy(selector: string): Chainable<any>
-    register(email: string, username: string, password: string): Chainable<any>
+    getByDataQa(selector: string): Chainable<any>;
+    getByDataCy(selector: string): Chainable<any>;
+    register(email: string, username: string, password: string): Chainable<any>;
+    login(email: string, username: string, password: string): Chainable<any>;
```

Comment:
The method 'login' takes three parameters which are 'email', 'username', and 'password'. Generally, user login processes only require 'email' and 'password'. Please verify if 'username' is needed here.